### PR TITLE
Feature/sensors alerts

### PR DESCRIPTION
**Pull request by LisaQ04**

### CODE REVIEW

File: waterlog/src/Pages/Usage.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React, { Component } from "react";
 import Usage from "./../Components/Usage";
+import TryCedric from "./../Components/TryCedric";
```
Comment (FredJEhlers) [inline waterlog/src/Pages/Usage.js]: What is this component? and Why is it a developer's name

File: waterlog/src/actions/AlertsAction.js [hunk 2]
```diff
@@ -35,3 +59,15 @@ export const fetchAlerts = (id) => dispatch => {
 		})
 		.catch(error => dispatch(fetchAlertsFailure(error)));
 };
+
+export const fetchFilteredAlerts =(segment, sensorId, sensortype, severity) => dispatch =>{ 
+	dispatch(fetchFilteredAlertsBegin());
+	fetch(`${process.env.REACT_APP_API_URL}/api/segmentevents/getalertsfilter?segment=${segment}&sensorId=${sensorId}&sensortype=${sensortype}&severity=${severity}`)
```
Comment (FredJEhlers) [inline waterlog/src/actions/AlertsAction.js]: split over two lines please

File: waterlog/src/actions/AlertsAction.js [hunk 3]
```diff
@@ -35,3 +59,15 @@ export const fetchAlerts = (id) => dispatch => {
...
+	.then(handleErrors)
+	.then(res => res.json())
+	.then(alerts=>{
+
```
Comment (FredJEhlers) [inline waterlog/src/actions/AlertsAction.js]: empty line not needed

File: waterlog/src/actions/Types.js [hunk 4]
```diff
@@ -143,6 +143,8 @@ export const FETCH_SEGMENT_BEGIN = "FETCH_SEGMENT_BEGIN";
 export const FETCH_SEGMENT_SUCCESS = "FETCH_SEGMENT_SUCCESS";
 export const FETCH_SEGMENT_FAILURE = "FETCH_SEGMENT_FAILURE";
 
+export const GET_PAGE_NUMBER = "GET_PAGE_NUMBER";
```
Comment (FredJEhlers) [inline waterlog/src/actions/Types.js:145]: move a line up

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Pages/Usage.js (hunk 1 above)
Comment (LisaQ04) [reply waterlog/src/Pages/Usage.js]: Removed dummy component.

File: waterlog/src/actions/AlertsAction.js (hunk 2 above)
Comment (LisaQ04) [reply waterlog/src/actions/AlertsAction.js]: Now split over multiple lines.

File: waterlog/src/actions/Types.js (hunk 4 above)
Comment (LisaQ04) [reply waterlog/src/actions/Types.js:145]: Removed empty line.

File: waterlog/src/actions/AlertsAction.js (hunk 3 above)
Comment (LisaQ04) [reply waterlog/src/actions/AlertsAction.js]: Removed line.

File: waterlog/src/actions/AlertsAction.js [hunk 5]
```diff
@@ -60,14 +60,18 @@ export const fetchAlerts = (id) => dispatch => {
...
-		dispatch(fetchFilteredAlertsSuccess(alerts));
-	})
-	.catch(error => dispatch(fetchFilteredAlertsFailure(error)))
-};
+export const fetchFilteredAlerts = (segment,
+	sensorId,
+	sensortype,
+	severity) => dispatch => {
+		dispatch(fetchFilteredAlertsBegin());
+		fetch(`${process.env.REACT_APP_API_URL}
```
Comment (FredJEhlers) [inline waterlog/src/actions/AlertsAction.js]: check your dm

File: waterlog/src/actions/AlertsAction.js [hunk 6]
```diff
@@ -60,14 +60,18 @@ export const fetchAlerts = (id) => dispatch => {
...
+	/api/segmentevents/getalertsfilter?segment=${segment}&sensorId=${sensorId}&sensortype=${sensortype}&severity=${severity}`)
+			.then(handleErrors)
+			.then(res => res.json())
+			.then(alerts => {
+
```
Comment (FredJEhlers) [inline waterlog/src/actions/AlertsAction.js]: remove empty line

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/actions/AlertsAction.js (hunk 5 above)
Comment (LisaQ04) [reply waterlog/src/actions/AlertsAction.js]: Fixed indentation.

File: waterlog/src/actions/AlertsAction.js (hunk 6 above)
Comment (LisaQ04) [reply waterlog/src/actions/AlertsAction.js]: Empty line is removed.

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**